taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -32,9 +32,11 @@ export const StartProject = () => {
 
   const findDefaultBranchName = useCallback(() => {
     if (taxonomyName === "" || ownerName === "") return "";
-    return `${toSnakeCase(taxonomyName.toLowerCase())}_${ownerName
-      .replace(" ", "")
-      .toLowerCase()}_${Math.floor(Date.now() / 1000)}`;
+    return `${toSnakeCase(
+      taxonomyName.toLowerCase()
```

Comment:
the `toSnakeCase` function already converts the argument to lower case, so the `.toLowerCase()` here is not needed